docs/api-guide.md

```diff
@@ -32,60 +36,71 @@ admin.close();
-|------------|----------------------------------------------|---------|
-| ru         | Base resource unit                           | 400     |
-| no-scaling | Disable auto-scaling for Cosmos DB for NoSQL | false   |
+<div id="Cassandra" class="tabcontent" markdown="1">
 
-For DynamoDB:
+| Name                 | Description                                                                            | Default          |
+|----------------------|----------------------------------------------------------------------------------------|------------------|
+| replication-strategy | Cassandra replication strategy. Must be `SimpleStrategy` or `NetworkTopologyStrategy`. | `SimpleStrategy` |
+| compaction-strategy  | Cassandra compaction strategy, Must be `LCS`, `STCS` or `TWCS`.                        | `STCS`           |
+| replication-factor   | Cassandra replication factor.                                                          | 1                |
 
-| name       | value                                  | default |
-|------------|----------------------------------------|---------|
-| no-scaling | Disable auto-scaling for DynamoDB      | false   |
-| no-backup  | Disable continuous backup for DynamoDB | false   |
-| ru         | Base resource unit                     | 10      |
+</div>
+<div id="Cosmos_DB_for_NoSQL" class="tabcontent" markdown="1">
 
-For Cassandra:
+| Name       | Description                                         | Default |
+|------------|-----------------------------------------------------|---------|
+| ru         | Base resource unit.                                 | 400     |
+| no-scaling | Disable auto-scaling for Cosmos DB for NoSQL.       | false   |
 
-| name                 | value                                                                                 | default          |
-|----------------------|---------------------------------------------------------------------------------------|------------------|
-| replication-strategy | Cassandra replication strategy, must be `SimpleStrategy` or `NetworkTopologyStrategy` | `SimpleStrategy` |
-| compaction-strategy  | Cassandra compaction strategy, must be `LCS`, `STCS` or `TWCS`                        | `STCS`           |
-| replication-factor   | Cassandra replication factor                                                          | 1                |
+</div>
+<div id="DynamoDB" class="tabcontent" markdown="1">
 
+| Name       | Description                             | Default |
+|------------|-----------------------------------------|---------|
+| no-scaling | Disable auto-scaling for DynamoDB.      | false   |
+| no-backup  | Disable continuous backup for DynamoDB. | false   |
+| ru         | Base resource unit.                     | 10      |
+
+</div>
+</div>
 
 ### Create a table
 
-Next, we will discuss table creation.
+When creating a table, you should define the table metadata and then create the table.
```

Comment:
> IMO, table metadata is not a commonly-defined word, so it would be better to explain what table metadata is in ScalarDB here.

@choplin Good idea! How about we use the following definition?:

> **Note**
> 
> The following is an excerpt from the **Table metadata** section in the blog post [ScalarDB Admin API & Schema Loader](https://medium.com/scalar-engineering/scalardb-admin-api-schema-loader-b0ad3809d4ea).

```suggestion
When creating a table, you should define the table metadata and then create the table.

Table metadata is ScalarDB-specific internal metadata for managing the schema of the user’s table. ScalarDB checks if the targeted metadata for the table exists and verifies that the name, type, and function of each column match the content in the `scalardb.metadata` database.
```

Reply:
Looks great! Thanks!